src/benchmarks/reconstruction/fecal_studies/CMakeLists.txt

```
@@ -0,0 +1,31 @@
+# Automatically set plugin name the same as the directory name
+# Don't forget string(REPLACE " " "_" PLUGIN_NAME ${PLUGIN_NAME}) if this dir has spaces in its name
+get_filename_component(PLUGIN_NAME ${CMAKE_CURRENT_LIST_DIR} NAME)
+print_header(">>>>   P L U G I N :   ${PLUGIN_NAME}    <<<<")       # Fancy printing
+
+# Function creates ${PLUGIN_NAME}_plugin and ${PLUGIN_NAME}_library targets
+# Setting default includes, libraries and installation paths
+plugin_add(${PLUGIN_NAME})
+
+# Find dependencies
+find_package(ROOT REQUIRED COMPONENTS Core Tree Hist RIO EG)
+plugin_add_event_model(${PLUGIN_NAME})
+plugin_add_dd4hep(${PLUGIN_NAME})
+plugin_add_acts(${PLUGIN_NAME})
+
+
+# The macro grabs sources as *.cc *.cpp *.c and headers as *.h *.hh *.hpp
+# Then correctly sets sources for ${_name}_plugin and ${_name}_library targets
+# Adds headers to the correct installation directory
+plugin_glob_all(${PLUGIN_NAME})
+
+# Add include directories
+# (same as target_include_directories but for both plugin and library)
+plugin_include_directories(${PLUGIN_NAME}
+        SYSTEM PUBLIC
+            ${ROOT_INCLUDE_DIRS})
+
+# Add libraries
+# (same as target_include_directories but for both plugin and library)
+#plugin_link_libraries(${PLUGIN_NAME} ${ROOT_LIBRARIES} ActsCore EDM4EIC::edm4eic)
+plugin_link_libraries(${PLUGIN_NAME} ${ROOT_LIBRARIES} ActsCore EDM4EIC::edm4eic algorithms_tracking)
```

```suggestion
plugin_link_libraries(${PLUGIN_NAME} ${ROOT_LIBRARIES} ActsCore EDM4EIC::edm4eic)
```
Just discovered, this breaks compilation on macOS. Tracking libs are not needed here.